doc/source/whatsnew/v0.16.0.txt

```diff
@@ -298,6 +298,82 @@ Backwards incompatible API changes
      p // 0
 
 
+Categorical Changes
+~~~~~~~~~~~~~~~~~~~
+
+.. _whatsnew_0160.api_breaking.categorical:
+
+In prior versions, Categoricals that with had an unspecified ordering (meaning no ``ordered`` keyword was passed) were defaulted to have a lexographic ordering of the values and designated as ``ordered`` Categoricals. Going forward, the ``ordered`` keyword in the ``Categorical`` constructor will default to ``False``, so ordering must now be explicit.
+
+Furthermore, previously you *could* change the ``ordered`` attribute of a Categorical by just setting the attribute, e.g. ``cat.ordered=True``; This is now deprecated and you should use ``cat.set_ordered(True)``. This will by default return a **new** object and not modify the existing object.
+
+Previous Behavior
+
+.. code-block:: python
+
+   In [1]: cat = pd.Categorical([0,1,2])
+
+   In [2]: cat
+   Out[2]:
+   [0, 1, 2]
+   Categories (3, int64): [0 < 1 < 2]
+
+   In [3]: cat.ordered
+   Out[3]: True
+
+   In [4]: cat.ordered=False
+
+   In [5]: cat
+   Out[5]:
+   [0, 1, 2]
+   Categories (3, int64): [0, 1, 2]
+
+
+New Behavior
+
+.. ipython:: python
+
+   cat = pd.Categorical([0,1,2])
+   cat
+   cat.ordered
+   cat.ordered=True
+   cat = cat.set_ordered(True)
+   cat
+   cat.ordered
+
+   # you can set in the construtor
+   cat = pd.Categorical([0,1,2],ordered=True)
+   cat
+   cat.ordered
+
+For ease of creation of series of ``Categoricals``, we have added the ability to pass keywords when calling ``.astype()``, these are passed directly to the constructor.
+
+.. ipython:: python
+
+   s = Series(["a","b","c","a"]).astype('category',ordered=True)
+   s
+   s = Series(["a","b","c","a"]).astype('category',categories=list('abcdef'),ordered=False)
+   s
+
+.. warning::
+
+   This simple API change may have suprising effects if a user is relying on the previous defaulted behavior implicity. In particular,
+   sorting operations with a ``Categorical`` will now raise an error:
+
+   .. code-block:: python
+
+      In [1]: df = DataFrame({ 'A' : Series(list('aabc')).astype('category'), 'B' : np.arange(4) })
+
+      In [2]: df['A'].order()
+      TypeError: Categorical not ordered
+      you can use .set_ordered(True) to change the Categorical to an ordered one
```

Comment:
the wording suggest that `set_ordered(True)` is inplace and it exposes the underlying object name instead of talking about "categorical data". On the other hand, I tried to get a better wording for 5 min, but couldn't find one :-(